css/style.css

```diff
@@ -1,19 +1,210 @@
-}
+/* phone size quary */
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+@media only screen and (max-width: 600px){ 
+    .body {
+        font-family: 'Roboto', sans-serif;
+        -webkit-font-smoothing: antialiased;
+        display: inline-block;
+        grid-template-rows: auto;
+        background-color: rgb(27, 196, 139);
+        }
+        
+        label {
+            display: block;
+            cursor: pointer;
+            width: 30px;
+            font-size: 20px;
+            float: right;
+            
+        }
+        .navigation{
+            display: grid;
+            grid-template-columns: 90% 10%;
+        }
 
+        .Nav a:hover, button:hover {
+            color: rgb(59, 25, 12);
+            font-style: italic;
+            font-weight: bolder;
+        }
+        #toggle{
+            visibility: hidden;
+        }
 
+        #toggle, .toggle{
+            grid-column: 2;
+            grid-row: 1;
+            position:relative;
+            
+        }
+        .Nav {
+            text-align: center;
+            width: 100%;
+            display: none;
+        }
+        .Nav a {
+            display: block;
+            border-bottom: 1px solid #EAEAEB;
+            padding: 10px;
+         
+        }
+        #toggle:checked + .Nav {
+            display: block;
+            text-align: center;
+        }
+
+ .logo {
+       width: 30px;
+       height: 30px;
+       padding-left: 15px;
+       position: relative;
+        }
+    
+    .backgroundimage {
+        width: 100%;
+        height: 50%;
+    }
+    .header {
+        width: 99%;
+        flex-direction: column;
+    }
+    .maintitle {
+       position: relative;
+       top: -100px;
+       text-align: center;
+       color: lavenderblush;
+       font-size: 20px;
+        }
+ .main {
+    align-content: center;
+    text-align: center;
+    position: relative;
+    top:-150px;
+      }
+ button {
+        background-color: green;
+        border-radius: 20%;
+        color: white;
+        padding: 5px;
+        width: auto;
+    }
+
+ .icons {
+        width: 13%;
+    }
+
+    .footer-icons {
+        width: 5%;
+        height: 5%px;
+    } 
+ .footer {
+        display: flexbox;
+        text-align: center;
+        position: relative;
+        top: -100%;
+         }
+  }
+
+  /* style and layout for bigger screens*/
+
+  @media screen and (min-width:601px) and (max-width: 2000px) {
+
+    .body {
+        font-family: 'Roboto', sans-serif;
+        -webkit-font-smoothing: antialiased;
+        display: grid;
+        grid-template-columns: auto;
+        background-color: rgb(140, 240, 140);
+        }
+    
```

Comment:
Nice work with making it all responsive! 

It's worth noting though that instead of having to repeat the same code for each `@media` block, you can have the stuff common for both designs outside of the `@media` blocks. This way `@media` will only be used for things that should be different on different sized screens